reactnative/README.md

```diff
@@ -0,0 +1,66 @@
+# React Native Input Mask
+
+### This is beta and is a still a WIP!
+
+## Getting started
+
+First, install it.
+
+```bash
+npm i react-native-text-mask --save
+```
+
+Then, require it and use it.
+
+```js
+import React, { Component } from 'react'
```

Comment:
Can you remove spacing around `Component` and add an ESLint rule for that please?!